manual/cypher/cypher-docs/src/docs/dev/ql/skip/index.asciidoc

```diff
@@ -3,6 +3,8 @@
 
 [abstract]
 `SKIP` defines from which row to start including the rows in the output.
+It takes a positive integer or an expression evaluating to a positive integer - however it is now allowed to refer to external identifiers.
```

Comment:
Should that be a `not` instead of `now`?
Also please use two dashes to produce the desired kind of dash in the output!
What is an `external identifier`?

Finally, I think this is too much detail for the abstract as we're using it.
Just move it to a paragraph on its own.